32blit-pico/CMakeLists.txt

```diff
@@ -45,15 +45,26 @@ if(${PICO_BOARD} STREQUAL "vgaboard")
 	)
 elseif(${PICO_BOARD} STREQUAL "pimoroni_picosystem")
 	message("Using picosystem...")
-	target_compile_definitions(BlitHalPico INTERFACE
-		AUDIO_BEEP_PIN=11
-		AUDIO_BEEP
-		DISPLAY_ST7789
-		INPUT_GPIO
-	)
+	if(AUDIO_PWM)
+		target_link_libraries(BlitHalPico INTERFACE pico_audio_pwm)
+		target_compile_definitions(BlitHalPico INTERFACE
+			PICO_AUDIO_PWM_MONO_PIN=11
+			PICO_AUDIO_PWM_PIO=1
+			AUDIO_PWM
+			DISPLAY_ST7789
+			INPUT_GPIO
+		)
+	else()
+		target_compile_definitions(BlitHalPico INTERFACE
+			AUDIO_BEEP_PIN=11
+			AUDIO_BEEP
+			DISPLAY_ST7789
+			INPUT_GPIO
+		)
+	endif()
```

Comment:
```cmake
	target_compile_definitions(BlitHalPico INTERFACE
		DISPLAY_ST7789
		INPUT_GPIO
	)

	if(AUDIO_PWM)
		target_link_libraries(BlitHalPico INTERFACE pico_audio_pwm)
		target_compile_definitions(BlitHalPico INTERFACE
			PICO_AUDIO_PWM_MONO_PIN=11
			PICO_AUDIO_PWM_PIO=1
			AUDIO_PWM
		)
	else()
		target_compile_definitions(BlitHalPico INTERFACE
			AUDIO_BEEP_PIN=11
			AUDIO_BEEP
		)
	endif()
```

Maybe? Doesn't make it any shorter, but uh... the duplication is annoying

(I have no idea how that file ended up using tabs when everything else is 2 spaces...)

Reply:
Agreed! We could even define the pins in all cases and just swap AUDIO_PWM/AUDIO_BEEP.

The `PICO_AUDIO_PWM_MONO_PIN` and `AUDIO_BEEP_PIN` should probably come from here somehow: 

https://github.com/raspberrypi/pico-sdk/blob/bfcbefafc5d2a210551a4d9d80b4303d4ae0adf7/src/boards/include/boards/pimoroni_picosystem.h#L52

Reply:
(Though there appears to be no good way to actually accomplish that)